src/csr.tex

```diff
@@ -194,16 +198,19 @@ \subsection*{CSR Access Ordering}
 the CSRs defined thus far in Volumes I and II of this specification are not
 directly accessible to other harts or devices and cause no side effects visible
 to other harts or devices.  Thus, accesses to CSRs other than the
-aforementioned three can be freely reordered with respect to FENCE instructions
+aforementioned three can be freely reordered in the global memory order
+with respect to FENCE instructions
 without violating this specification.
 \end{commentary}
 
-For CSR accesses that cause side effects, the above ordering constraints apply
-to the order of the initiation of those side effects but does not necessarily
-apply to the order of the completion of those side effects.
-
```

Comment:
Should this be left in place until the "rules for the reordering of CSR accesses in the global memory order" are added to the RVWMO chapter.  Maybe with a note that this statement is part of what should be moved.